### ISSUE TITLE
Loss Is Not Decreasing When Training ViViT

### ISSUE THREAD

**Issue by UsmanNiz** [mx-mark/VideoTransformer-pytorch#39]
Loss remains the same when training ViViT, I have tried mulitple hyperparametrs changes but nothing seems to work for me.
